packages/cli/src/commands/connector/utils.ts

```diff
@@ -232,16 +232,21 @@ const fetchOfficialConnectorList = async () => {
   return packages;
 };
 
-export const addOfficialConnectors = async (instancePath: string) => {
+export const addOfficialConnectors = async (
+  instancePath: string,
+  excludePackageNames: string[] = []
```

Comment:
maybe we can simply put a boolean here, and pass into `fetchOfficialConnectorList`, then update `packages/cli/src/commands/connector/utils.ts:220` to conditionally add `'cloud'` using `conditionalArray()`

<img width="399" alt="image" src="https://user-images.githubusercontent.com/14722250/225206127-5c29b8b2-5fa9-4da6-a231-4a2eb548baa1.png">

Reply:
updated